plugins/modules/zabbix_screen.py

```diff
@@ -302,7 +290,7 @@ def delete_screen_items(self, screen_id, screen_item_id_list):
                 self._zapi.screenitem.delete(screen_item_id_list)
                 return True
             return False
-        except ZabbixAPIException:
+        except Exception:
```

Comment:
This may be against "best practices"

Reply:
You're right. I don't like the two approaches either. I'm still on vacation, keep it open I will see how I can get this into the Wrapper class or a helper.

Reply:
No worries, no rush! Thanks for all of your work tho :)

How bad of an idea is to just import into `module_utils.wrapper`, not use it, but then call it from modules like so:
`from ansible_collections.community.zabbix.plugins.module_utils.base import ZabbixBase, Already_Exists, ZabbixAPIException`?

But this will probably complain in linting or tox (since we will have that as well).

Reply:
I've now implemented a class with static functions to wrap around the Already_Exists exception.
On the other hand I've still imported ZabbixAPIException from wrappers.py as you suggested... but to be true I don't understand why we `pass` the exception and don't exit with an error if the ScreenItem can't be deleted?

Reply:
I am not very familiar with this modules to honest and I haven't thought about it when reviewing as only thing I've considered was a diff itself. If you are up for it, this can be changed. Raising exceptions in this context is stupid anyways. This is one of the modules that would really need rework